clang/lib/Driver/Driver.cpp

```diff
@@ -751,6 +752,21 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
                 CudaTC = std::make_unique<toolchains::CudaToolChain>(
                     *this, TT, *HostTC, C.getInputArgs(), Action::OFK_OpenMP);
               TC = CudaTC.get();
+            } else if (TT.isAMDGCN()) {
+              const ToolChain *HostTC =
+                  C.getSingleOffloadToolChain<Action::OFK_Host>();
+              const llvm::Triple &HostTriple = HostTC->getTriple();
+              StringRef DeviceTripleStr;
+              DeviceTripleStr = "amdgcn-amd-amdhsa";
+              llvm::Triple HIPTriple(DeviceTripleStr);
```

Comment:
This looks like it can be HIPTriple("amdgcn-amd-amdhsa"), and we should call it something other than HIP